resource-graph-patterns/config-pattern/resource-definitions/s3-terraform.exclude

```diff
@@ -0,0 +1,47 @@
+apiVersion: entity.humanitec.io/v1b1
```

Comment:
Suggesting to add a quick explanation of this file as it is not mentioned in the README:

```suggestion
# The example s3 Resource Definition would look like this when using the Terraform Driver
# instead of the Echo Driver to actually provision an s3 resource.
apiVersion: entity.humanitec.io/v1b1
```